### PR TITLE
Removed fields strict validation

### PR DESCRIPTION
**Pull request by ramsharma-prog**
- The answer validation failed due to a missing answer value not counting as nullable.

### CODE REVIEW

**Comment by gidsg** [comment]
@Adam-W1 agree we should test this, but maybe create a follow-up card for that, so we can get this hotfix out?

File: openapi/components.yml [hunk 1]
```diff
@@ -78,7 +78,6 @@ components:
           properties:
             application:
               type: object
-              additionalProperties: true
```
Comment (gidsg) [inline openapi/components.yml:81]: Do we definitely want to remove this? Might leaving this make us more defensive to changes in future?

**Comment by Adam-W1** [comment]
> @Adam-W1 agree we should test this, but maybe create a follow-up card for that, so we can get this hotfix out?

Sounds good ill create the card

File: openapi/components.yml (hunk 1 above)
Comment (ramsharma-prog) [reply openapi/components.yml:81]: We tested by adding additional keys to meta data & application level. The validation doesn't care about additional keys

**Review by gidsg — APPROVED** [review]
LGTM 👍

**Pull request merged**